.github/workflows/test-action.yml

```diff
@@ -1,6 +1,6 @@
 name: test-cosign
 
-on: [pull_request]
+on: [pull_request, workflow_dispatch]
```

Comment:
can you remove this extra trigger?
maybe add the 
```
on:
  push:
     - main
```

Reply:
Ah yeah I forgot to explain this in my PR originally. I added this to make it easier for me to run these tests since I was changing the inputs to one of them. Your suggestion would also work. I will update this.

Reply:
I have updated this to `on push main` as you suggested